src/js/claims-status/containers/ClaimsStatusApp.jsx

```diff
@@ -28,9 +29,11 @@ class ClaimsStatusApp extends React.Component {
         userProfile={this.props.profile}
         loginUrl={this.props.loginUrl}
         verifyUrl={this.props.verifyUrl}>
-        <AppContent>
-          {this.props.children}
-        </AppContent>
+        <DowntimeNotification appTitle="claims status tool" dependencies={[services.evss]}>
```

Comment:
This is going to block someone from viewing their appeals if evss is down. Is that ok?

Reply:
Not sure, is there a better place to add the Downtime Notification? I went purely based on the spreadsheet and what seemed to me as the best entry points but feel free to point me in the right direction if I was off on any of this.

Reply:
This app has two independent dependencies, so given the current design, I don't think there's a good place to put it that doesn't block use of the app if just one of the dependencies is down.

Reply:
The downtime notification doesn't have to wrap the whole thing - is it blocking a menu in this case? And that's why they wouldn't be able to view their appeals if evss is down?

Reply:
In the claim status app the initial page of the app is a list of claims and appeals, mixed together. My understanding is that this would replace that whole page with the downtime message.

Reply:
Yeah, this is one of those places that an all-or-nothing block isn't a good fit. @ncksllvn here's kind of how it works:

The "claims-status" app is a bit of a misnomer. It acts as a way for the Veteran to view their claims _and_ the appeals to those claims (which are often seen as part of the claims themselves, which is why we've kept the name). When we go to the app, it'll query for both claims and appeals and list them on the index page. After one is selected, it'll open up either the claims status page or the appeals status page, depending on the type.

The trouble here is that claims and appeals rely on two different services. We could block those status pages since they rely on one service each, but that doesn't give the user any warning that a maintenance window has been scheduled. Ideally, we'd be able to handle showing messages to indicate either claims or appeals are down, and block the whole app if they're both down.

But that's not MVP.

@djmunoz1 or @rroueche, does it make sense to have this downtime notification block the app if _either_ evss or the appeals service is down?